deb/deb.go

```diff
@@ -576,14 +533,14 @@ func createControl(instSize int64, md5sums []byte, info *nfpm.Info) (controlTarG
 	}
 
 	filesToCreate := map[string][]byte{
-		"control":   body.Bytes(),
-		"md5sums":   md5sums,
-		"conffiles": conffiles(info),
+		"./control":   body.Bytes(),
+		"./md5sums":   md5sums,
+		"./conffiles": conffiles(info),
```

Comment:
I don't quite understand why this change is needed... could you add some details?